Microsoft.Toolkit.Uwp.SampleApp/SamplePages/samples.json

```diff
@@ -759,6 +759,14 @@
         "About": "The ThemeListener allows you to keep track of changes to the System Theme.",
         "Icon": "/Assets/Helpers.png",
         "DocumentationUrl": "https://raw.githubusercontent.com/MicrosoftDocs/WindowsCommunityToolkitDocs/master/docs/helpers/ThemeListener.md"
+      },
+      {
+        "Name": "OnDevice",
+        "Type": "OnDevicePage",
+        "About": "The OnDevice markup extension allows you to customize UI appearance on a per-DeviceFamily basis.",
+        "Icon": "/SamplePages/OnDevice/OnDevice.png",
+        "XamlCodeFile": "OnDeviceXaml.bind",
+        "DocumentationUrl": "https://raw.githubusercontent.com/MicrosoftDocs/WindowsCommunityToolkitDocs/master/docs/helpers/OnDevice.md"
```

Comment:
Be sure to open a doc PR here as well, eh? https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs

Reply:
Agree. I have changed the code.

Reply:
Doc PR: https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/pull/158